### PR TITLE
feat: remove product specific rules, adjust web rules

### PR DESCRIPTION
**Pull request by romenkova**

### CODE REVIEW

File: lib/config/sharedConfig.js [hunk 1]
```diff
@@ -49,52 +49,12 @@ module.exports = {
...
     },
-    // next: {
-    //   rootDir: "/projects/*/",
-    // },
     "import/core-modules": ["react"],
     "import/parsers": {
       "@typescript-eslint/parser": [".ts", ".tsx"],
     },
     "import/extensions": [".js", ".ts", ".tsx"],
   },
```
Comment (romenkova) [inline lib/config/sharedConfig.js:57]: moved to separate variable

File: lib/config/sharedConfig.js [hunk 2]
```diff
@@ -143,44 +103,6 @@ module.exports = {
         ignoreRegExpLiterals: true,
       },
     ],
-    "no-restricted-imports": [
```
Comment (romenkova) [inline lib/config/sharedConfig.js:146]: these are hardly connected to the project itself, specifically web, I moved to web monorepo config

File: lib/config/sharedConfig.js [hunk 3]
```diff
@@ -399,12 +321,6 @@ module.exports = {
     "@typescript-eslint/prefer-optional-chain": ["warn"],
     "prefer-regex-literals": ["warn"],
     "import/no-import-module-exports": ["warn"],
-    "deprecate/import": [
```
Comment (romenkova) [inline lib/config/sharedConfig.js:402]: moved to web monorepo config as well

File: lib/rules/shared.js [hunk 4]
```diff
@@ -46,44 +46,6 @@ module.exports = {
       ignoreRegExpLiterals: true,
     },
   ],
-  "no-restricted-imports": [
```
Comment (romenkova) [inline lib/rules/shared.js:49]: moved to web monorepo config

File: lib/rules/shared.js [hunk 5]
```diff
@@ -274,8 +236,7 @@ module.exports = {
       "newlines-between": "never",
     },
   ],
-  "@next/next/no-img-element": 0,
-  "@next/next/no-html-link-for-pages": "off",
```
Comment (romenkova) [inline lib/rules/shared.js:278]: this is web-specific, doesn't belong to shared

File: lib/rules/shared.js [hunk 6]
```diff
@@ -284,11 +245,5 @@ module.exports = {
   "@typescript-eslint/prefer-optional-chain": ["warn"],
   "prefer-regex-literals": ["warn"],
   "import/no-import-module-exports": ["warn"],
-  "deprecate/import": [
```
Comment (romenkova) [inline lib/rules/shared.js:287]: moved to web monorepo config

**Review by ddiazFG — APPROVED** [review]

**Pull request merged**